cmd/kubelet/app/server.go

```diff
@@ -273,10 +273,8 @@ is checked every 20 seconds (also configurable with a flag).`,
 			}
 
 			if utilfeature.DefaultFeatureGate.Enabled(zpagesfeatures.ComponentFlagz) {
-				if cleanFlagSet != nil {
-					namedFlagSet := map[string]*pflag.FlagSet{server.ComponentKubelet: cleanFlagSet}
-					kubeletDeps.Flagz = flagz.NamedFlagSetsReader{FlagSets: cliflag.NamedFlagSets{FlagSets: namedFlagSet}}
-				}
```

Comment:
I'd keep it as is to ensure that even if it's changed to nil somewhere we're still safe.

Reply:
It shouldn't be set to nil.

https://github.com/kubernetes/kubernetes/blob/d019f301d5a2d63a3af304e5557235af4d076adf/cmd/kubelet/app/server.go#L140